Clarify which contributing docs refer to what (#12384)
microsoft/fluentui: .github/CONTRIBUTING.md

> # Contributing to `@fluentui/react-northstar` (and others under `packages/fluentui`)

Reviewer: Should we move this file to `packages/fluentui`, too?

  - [Keyboard handling](#keyboard-handling)
- [Packages](#packages)
  - [Add a new package](#add-a-new-package)
    - [Run `syncpack format`](#run-syncpack-format)
  - [Add a new dependency](#add-a-new-dependency)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Getting started

Make sure you have [Node.js][1] version 10 or later installed.

**If you would like to contribute to packages besides the ones under `packages/fluentui`, see [this wiki page](https://github.com/microsoft/fluentui/wiki/Contributing) instead!**

### Useful Commands

> This list contains the most useful commands. You should run `yarn run` to see all scripts.

From repo root:

```sh

> yarn start                 # start doc site: choose `@fluentui/docs` for v0

Reviewer 2: ```suggestion
yarn start                 # start doc site: choose `@fluentui/docs` for `@fluentui/react-northstar`
```

yarn test                  # run all packages' tests once

yarn build                 # build all packages
yarn build:fluentui:docs   # build docs

gulp deploy:docs           # deploy gh-pages doc site

yarn lint                  # lint all packages

yarn prettier              # run prettier on changed files
```

From an individual under `packages/fluentui`, such as `packages/fluentui/react-northstar`:

```sh
yarn test                  # test once
yarn test:watch            # test on file change

yarn build                 # build all files in package

yarn lint                  # lint once
yarn lint:fix              # lint and attempt to fix
```

## Workflow

These guides will walk your through various activities for contributing:

- [Set up Local Development](setup-local-development.md)
- [Add a feature](add-a-feature.md)
- [Test a feature](test-a-feature.md)
- [Document a feature](document-a-feature.md)
- [Release a new package version](release-a-package.md)

## Accessibility

> Fluent UI v0 implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-\* properties) as well as handle keyboard interaction and focus. Every accessible component has a default behavior, which can be overridden using the `accessibility` prop. You can choose a behavior from the ones provided by Fluent UI or you can implement a new behavior.

Reviewer 2: ```suggestion
`@fluentui/react-northstar` implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-\* properties) as well as handle keyboard interaction and focus. Every accessible component has a default behavior, which can be overridden using the `accessibility` prop. You can choose a behavior from the ones provided by Fluent UI or you can implement a new behavior.
```

An application should always have an element with [focus][7] when in use. The user can change the focused element by:

- pressing tab/shift+tab keys to navigate through the components
- pressing arrow keys to navigate through children (for example menu items in menu)
- using the screen reader with or without virtual cursor

> Fluent UI v0 uses the [FocusZone][8] component for basic tab and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).

Reviewer 2: ```suggestion
Fluent UI uses the [FocusZone][8] component for basic tab and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).
```

> Focused component needs to be clearly visible. This is handled in Fluent UI v0 by focus indicator functionality. Focus indicator will be displayed only if the application is in keyboard mode. Application switches to keyboard mode when a key relevant to navigation is pressed. It disables keyboard mode on mouse click events.

Reviewer 2: ```suggestion
Focused component needs to be clearly visible. This is handled in `@fluentui/react-northstar` by focus indicator functionality. Focus indicator will be displayed only if the application is in keyboard mode. Application switches to keyboard mode when a key relevant to navigation is pressed. It disables keyboard mode on mouse click events.
```

### Add a new package

<!--
#### Run `lerna create`

You should to run `lerna create` command to create a new package

> -->

Reviewer: ```suggestion
```

  name: '__DIRECTORY_NAME__',
};
```

> -->

Reviewer: Let's also remove this section (currently commented)
